doc/rados/command/list-inconsistent-obj.json

```diff
@@ -0,0 +1,140 @@
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
+            "description": "Identify a Ceph object",
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "nspace": {
+                "type": "string"
+              },
+              "locator": {
+                "type": "string"
+              },
+              "snap": {
+                "oneOf": [
+                  {
+                    "type": "string",
+                    "enum": [ "head", "snapdir" ]
+                  },
+                  {
+                    "type": "integer",
+                    "minimum": 0
+                  }
+                ]
+              }
+            },
+            "required": [
+              "name",
+              "nspace",
+              "locator",
+              "snap"
+            ]
+          },
+          "errors": {
+            "description": "Union of all shard errors, including errors in which the faulty shard is NOT identified",
+            "type": "array",
+            "items": {
+              "enum": [
+                "attr_unexpected",
+                "missing",
+                "stat_error",
+                "read_error",
+                "data_digest_mismatch",
+                "omap_digest_mismatch",
+                "size_mismatch",
+                "attr_mismatch",
+                "attr_missing",
+                "data_digest_mismatch_oi",
+                "omap_digest_mismatch_oi",
+                "size_mismatch_oi"
+              ]
+            },
+            "minItems": 0,
+            "uniqueItems": true
+          },
+          "shards": {
+            "description": "All found or expected shards",
+            "type": "array",
+            "items": {
+              "description": "Information about a particular shard of object",
+              "type": "object",
+              "properties": {
+                "osd": {
+                  "type": "integer"
+                },
+                "size": {
+                  "type": "integer"
+                },
+                "omap_digest": {
+                  "description": "Hex representation (e.g. 0x1abd1234)",
+                  "type": "string"
+                },
+                "data_digest": {
+                  "description": "Hex representation (e.g. 0x1abd1234)",
+                  "type": "string"
+                },
+                "errors": {
+                  "description": "Errors that have been definitively identified as belonging to this shard",
+                  "type": "array",
+                  "items": {
+                    "enum": [
+                      "attr_unexpected",
+                      "missing",
+                      "stat_error",
+                      "read_error",
+                      "data_digest_mismatch",
+                      "omap_digest_mismatch",
```

Comment:
I don't think these would be valid here.  Either it would have to be present on all shards, or we'd be making a guess as to which are correct.  I don't think that makes sense since the above error set already indicates that there is a mismatch and we have the actual hash values for each shard.